src/hotspot/share/classfile/placeholders.cpp

```diff
@@ -31,7 +31,26 @@
 #include "memory/resourceArea.hpp"
 #include "runtime/javaThread.hpp"
 #include "runtime/mutexLocker.hpp"
-#include "utilities/hashtable.inline.hpp"
+#include "utilities/resourceHash.hpp"
+
+class PlaceholderKey : public StackObj {
```

Comment:
Since the PlaceholderKey is stored inside the _placeholders table, I think it shouldn't be declared as StackObj.

Reply:
Ok, fixed.